### DISCUSSION TITLE
Join and share your funny moments! 😄

### DISCUSSION THREAD

**Discussion by skydoves** [GetStream/AvengersChat#1] [Announcements]
If you find some funny moments on this demo application, feel free to join and share that moment in this thread!

<p align="center">
<img src="https://user-images.githubusercontent.com/24237865/138444812-8a9eea62-5543-42f6-96bb-635fa365b3ab.jpg" width="32%"/>
<img src="https://user-images.githubusercontent.com/24237865/138444814-2cddcecc-e5ed-4d3f-8aca-6148cf315bb4.jpg" width="32%"/>
<img src="https://user-images.githubusercontent.com/24237865/138444817-04f6f833-4856-4733-aab2-dd3d7223093d.jpg" width="32%"/>
</p>

**Comment by masterewot** [comment]
Interesting. I have many funny moments in my life!
